lib/elixir/test/elixir/enum_test.exs

```diff
@@ -1861,6 +1861,7 @@ defmodule EnumTest.Range do
   test "map/2" do
     assert Enum.map(1..3, fn x -> x * 2 end) == [2, 4, 6]
     assert Enum.map(-1..-3, fn x -> x * 2 end) == [-2, -4, -6]
+    assert Enum.map(1..10//2, fn x -> x * 2 end) == [2, 6, 10, 14, 18]
```

Comment:
Can you add tests for negative steps and the cases that return empty lists?

Reply:
Great point, added in [88d85c9](https://github.com/elixir-lang/elixir/pull/12554/commits/88d85c9ec7e0300e47d0d7861c79ed186edf9870).